JSON_for_IO-Link.yaml

```diff
@@ -4974,6 +4974,8 @@ components:
       enum:
         - MANUAL
         - DHCP
+        - AUTO_IP
+        - DCP
```

Comment:
is DCP here at the right place ? DCP is a service and not a configuration.

Reply:
all of this was already exiting in the released specification.

Reply:
this is a subject to be changed in the future.

Reply:
a description for the added enum values would be good